packages/web3-eth/src/signers/TransactionSigner.js

```diff
@@ -70,13 +71,15 @@ export default class TransactionSigner {
 
         const rlpEncoded = ethTx.serialize().toString('hex');
         const rawTransaction = '0x' + rlpEncoded;
+        const transactionHash = Hash.keccak256(rawTransaction)
```

Comment:
Use web3-utils sha3 here instead of bringing in eth-lib dependency!

Reply:
Fixed it! please take a look again 😄